themes/hugo-asyncapi/layouts/partials/docs/content.html

```diff
@@ -8,5 +8,15 @@ <h1 class="title section-title">{{ .Title }}</h1>
         {{- with .Content -}}
         {{ . | replaceRE "(<h[1-9] id=\"([^\"]+)\".+)(</h[1-9]+>)" `${1}<a href="#${2}" class="hanchor" ariaLabel="Anchor"> #&#xFE0E;</a> ${3}` | safeHTML }}
         {{- end -}}
+        <div class="hint">
```

Comment:
I love the idea of the PR but use the `hint` shortcode. Like this:

```markdown
{{%hint%}}
Improve this page...
{{%/hint%}}
```

If you need to put HTML inside, use this instead:

```html
{{<hint>}}
Your HTML content here...
{{</hint>}}
```

Reply:
Also, you can probably move this piece of content to another file and just include it right after [this line](https://github.com/asyncapi/website/blob/master/themes/hugo-asyncapi/layouts/docs/single.html#L9).

Reply:
> use the hint shortcode
According to https://gohugo.io/content-management/shortcodes/#what-a-shortcode-is shortcodes cannot be used in templates, so I can't use it in partial as build fails.

> Also, you can probably move this piece of content to another file and just include it right after this line.

If I place my code where you suggest it looks like outside the content, gets container's width, which I'm not sure is good. The advantage is that it is more prominent to the viewer I think. What do you think

<img width="1457" alt="Screenshot 2020-02-12 at 11 08 56" src="https://user-images.githubusercontent.com/6995927/74324952-216e7700-4d88-11ea-9698-55d9b01a2a2b.png">


For now I placed it in the same `div` as the content, and moved HTML to a separate partial - have a look in the preview

Reply:
🤔 I think we can probably leave it there but we should change the appearance so it doesn't look broken. Maybe removing the question mark, the veritcal line, and assigning a gray color to the text would work. What do you think?

Reply:
Something like this:

<img width="1193" alt="Screen Shot 2020-02-12 at 12 18 04" src="https://user-images.githubusercontent.com/242119/74330439-fee15b80-4d91-11ea-8a5f-96e06f5b80a7.png">